Automattic/vip-go-mu-plugins
# Trigger a warning for prolonged rate limiting

## Description

Presently we just push the messages to Slack. We should also trigger a warning so functionality built around that can pick it up and put it where it needs to be.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Set the rate limiting thresholds for Search queries and indexing low.
2. Trigger ratelimiting. There should be no warning generated.
3. Apply PR.
4. Trigger ratelimiting. A warning should be generated.
